helm/ffc-workstream/values.yaml

```diff
@@ -0,0 +1,15 @@
+name: ffc-demo
+namespace: ffc-demo
+workstream: ffc-demo
+environment: production
+
+labels:
+  instance:
+  version: 1.0.0
+  component: ffc-demo
```

Comment:
According to https://eaflood.atlassian.net/wiki/spaces/FPS/pages/1618214984/Kubernetes+labels this should be `service` (which would also make it match all the other ffc demo service k8s objects). You could even omit this if you use the library template for labels in `resourceQuota.yaml` as it will set the component label to `service` by default.